notebooks/00_quick_start/dkn_single_node.ipynb

```diff
@@ -0,0 +1,243 @@
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_path = '../../tests/resources/deeprec/dkn'\n",
+    "yaml_file = os.path.join(data_path, r'dkn.yaml')\n",
+    "train_file = os.path.join(data_path, r'final_train_with_entity.txt')\n",
+    "valid_file = os.path.join(data_path, r'final_test_with_entity.txt')\n",
+    "wordEmb_file = os.path.join(data_path, r'word_embeddings_100.npy')\n",
+    "entityEmb_file = os.path.join(data_path, r'TransE_entity2vec_100.npy')\n",
+    "if not os.path.exists(yaml_file):\n",
+    "    download_deeprec_resources(r'https://recodatasets.blob.core.windows.net/deeprec/', data_path, 'dknresources.zip')\n",
+    "    \n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create hyper-parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[('DNN_FIELD_NUM', None), ('FEATURE_COUNT', None), ('FIELD_COUNT', None), ('MODEL_DIR', None), ('PAIR_NUM', None), ('SUMMARIES_DIR', None), ('activation', ['sigmoid']), ('attention_activation', 'relu'), ('attention_dropout', 0.0), ('attention_layer_sizes', 100), ('batch_size', 100), ('cross_activation', 'identity'), ('cross_l1', 0.0), ('cross_l2', 0.0), ('cross_layer_sizes', None), ('cross_layers', None), ('data_format', 'dkn'), ('dim', 100), ('doc_size', 10), ('dropout', [0.0]), ('dtype', 32), ('embed_l1', 0.0), ('embed_l2', 0.0005), ('enable_BN', False), ('entityEmb_file', '../../tests/resources/deeprec/dkn\\\\TransE_entity2vec_100.npy'), ('entity_dim', 100), ('entity_embedding_method', 'TransE'), ('entity_size', 3777), ('epochs', 10), ('fast_CIN_d', 0), ('filter_sizes', [1, 2, 3]), ('init_method', 'uniform'), ('init_value', 0.1), ('is_clip_norm', 0), ('iterator_type', None), ('kg_file', None), ('kg_training_interval', 5), ('layer_l1', 0.0), ('layer_l2', 0.0005), ('layer_sizes', [300]), ('learning_rate', 2e-05), ('load_model_name', None), ('load_saved_model', False), ('loss', 'log_loss'), ('lr_kg', 0.5), ('lr_rs', 1), ('max_grad_norm', 2), ('method', 'classification'), ('metrics', ['auc', 'acc', 'f1', 'logloss']), ('model_type', 'dkn'), ('mu', None), ('n_item', None), ('n_item_attr', None), ('n_user', None), ('n_user_attr', None), ('num_filters', 100), ('optimizer', 'adam'), ('reg_kg', 0.0), ('save_epoch', 2), ('save_model', False), ('show_step', 100000), ('train_ratio', None), ('transform', True), ('use_CIN_part', False), ('use_DNN_part', False), ('use_FM_part', False), ('user_Linear_part', False), ('user_clicks', None), ('user_dropout', False), ('wordEmb_file', '../../tests/resources/deeprec/dkn\\\\word_embeddings_100.npy'), ('word_size', 8033), ('write_tfevents', False)]\n"
+     ]
+    }
+   ],
+   "source": [
+    "hparams = prepare_hparams(yaml_file, wordEmb_file=wordEmb_file, entityEmb_file=entityEmb_file, epochs=10)\n",
```

Comment:
I think it would be interesting to give some information about the hyperparameters

Reply:
OK. I will add comments in the corresponding yaml file.